src/core/public/chrome/ui/header/header_breadcrumbs.scss

```diff
@@ -0,0 +1,35 @@
+/*
+* Copyright OpenSearch Contributors
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+.headerBreadcrumbs {
```

Comment:
Is this going to move to OUI? Shouldn't this be a variant in oui that gets turned on based on a flag?

Reply:
We are thinking of a brand new component and marking the old one as deprecated.